Add anoncreds issuance and presentation support [DEV-4592]

DaevMithran opened the pull request.

jamshale committed:
Add anoncreds issuance and presentation formats

Signed-off-by: jamshale <jamiehalebc@gmail.com>

jamshale committed:
Fix unit tests

Signed-off-by: jamshale <jamiehalebc@gmail.com>

jamshale committed:
Fix error in _formats_attach

Signed-off-by: jamshale <jamiehalebc@gmail.com>

jamshale committed:
Fix AnoncredsDID validation pattern

Signed-off-by: jamshale <jamiehalebc@gmail.com>

jamshale committed:
Fix name

Signed-off-by: jamshale <jamiehalebc@gmail.com>

jamshale committed:
Fix scenario test

Signed-off-by: jamshale <jamiehalebc@gmail.com>

jamshale committed:
Update demo with anoncreds format

Signed-off-by: jamshale <jamiehalebc@gmail.com>

jamshale committed:
Fix the scenario test and add better description

Signed-off-by: jamshale <jamiehalebc@gmail.com>

jamshale committed:
Add service_healthy check for indy agent

Signed-off-by: jamshale <jamiehalebc@gmail.com>

jamshale committed:
Fix/update BDD tests

Signed-off-by: jamshale <jamiehalebc@gmail.com>

jamshale committed:
Add issuer askar to holder anoncreds support

Signed-off-by: jamshale <jamiehalebc@gmail.com>

jamshale committed:
Update BDD tests

Signed-off-by: jamshale <jamiehalebc@gmail.com>

jamshale committed:
Remove commented code

Signed-off-by: jamshale <jamiehalebc@gmail.com>

jamshale committed:
Expand scenario test

Signed-off-by: jamshale <jamiehalebc@gmail.com>

jamshale committed:
Add better comments for format compatibilty imports

Signed-off-by: jamshale <jamiehalebc@gmail.com>

DaevMithran committed:
Merge anoncreds issuance and presentation

ankurdotb commented:
Task linked: [DEV-4592 Complete credential issuance flow with ACA-Py](https://app.clickup.com/t/6600954/DEV-4592)

DaevMithran committed:
Merge branch 'acapy-cheqd' into issuance-presentation

Commit:
Updated tests

Signed-off-by: Sownak Roy <sownak@cheqd.io>

DaevMithran committed:
Support cheqd identifiers in issuance

DaevMithran committed:
Init did:cheqd tests

DaevMithran committed:
Update scenarios

Commit:
make image build on ARM64

Commit:
fix example

Commit:
split main func

Commit:
add create schema

Commit:
add more cases

DaevMithran committed:
Add issuance test

DaevMithran committed:
test: add integration tests [DEV-4613][DEV-4614] (#4)

Changes:
- extend integration tests:
    - create schema
    - create credential definition
- assert credential definition available in credential definitions list
    - assert did available in wallet dids
    - deactivate did

jamshale committed:
Use entropy instead of prover_did

Signed-off-by: jamshale <jamiehalebc@gmail.com>

DaevMithran committed:
Use entropy instead of prover_did (#6)

DaevMithran committed:
fix auto issue
